.gitignore

```diff
@@ -1 +1,3 @@
-.idea/
\ No newline at end of file
+.idea/
+out/
+Main.java
```

Comment:
Please add a new line at the end of this file

Reply:
Done